qa/suites/fs/upgrade/featureful_client/old_client/branch/n-1.yaml

```diff
@@ -0,0 +1,2 @@
+teuthology:
+  branch: squid
```

Comment:
This suite checks if an upgraded mds can evict older clients after the fs is set with `required_client_features`, specifically `metric_collect`. So, octopus client does not support `metric_collect` and the test verifies this.

So, I don't think this needs to be touched. I expect the tests to fail with pacific+ since those release support `metric_collect` feature and the MDS will not evict those clients thereby resulting in failed tests.